src/unitxt/metrics.py

```diff
@@ -2134,11 +2134,10 @@ def compute(
         return self.pipe(predictions, batch_size=self.batch_size)
 
 
-class LlamaIndexCorrectness(InstanceMetric):
-    """LlamaIndex based metric class for evaluating correctness."""
-
+class LlamaIndexLLMMetric(InstanceMetric):
     model_name: str = ""
     main_score: str = ""
+    metric_name_prefix: str = ""
```

Comment:
There is now a generic score_prefix in Metric base class.  Maybe it can be used, instead of adding this new name ?

#
    # Used to add a prefix to all score, except the "score_name" and "score" fields.
    # This is used to distinguish two scores of the same metrics, operating on different fields of the task
    #
    score_prefix: str = ""

Reply:
Sure, I wasn't aware of it